lab3/lab3/lab3.cpp

```diff
@@ -0,0 +1,102 @@
+#include <mpi.h>
+#include <iostream>
+#include <string>
+#include <vector>
+
+int procId;
+int numProcs;
+
+struct student
+{
+    int id;
+    char name[10];
+    char surname[10];
+    student(int id, const std::string &name, const std::string &surname)
+    {
+        this->id = id;
+        memset(&this->name[0], 0, 10);
+        memset(&this->surname[0], 0, 10);
+        strcpy_s(this->name, name.c_str());
```

Comment:
This constructor is not needed anymore, but you should have used `std::strcpy` instead